src/js/components/Banner.jsx

```diff
@@ -0,0 +1,26 @@
+import {bannerText} from '../../../config'
+import {grey800, yellow200} from 'material-ui/styles/colors'
+import React, {Component} from 'react'
+
+const style = {
+  banner: {
+    background: grey800,
+    color: yellow200,
+    fontSize: '0.8rem',
+    fontWeight: 400,
+    textAlign: 'center'
+  }
+}
+
+export class Banner extends Component {
```

Comment:
Yeah. so functional components are functions that return jsx. a regular component (class based) is a constructor and has lifecycle methods attached. in the future, the react team has said there will likely be optimizations that can only be made for functional components. so we should go with them as much as we can. if a component doesn't use lifecycle methods, than it should just be a functional component.